ding/envs/env_wrappers/env_wrappers.py

```diff
@@ -306,31 +332,53 @@ def step(self, action):
 class DelayRewardWrapper(gym.Wrapper):
     """
     Overview:
-        Return cumulative reward at intervals; At other time, return reward of 0.
-    Interface:
-        ``__init__``, ``reset``, ``step``, ``new_shape``
+        The DelayRewardWrapper class is a gym wrapper that delays the reward. It cumulates the reward over a
```

Comment:
which aims to simulate sparse reward environment

Reply:
polished